zenodo/modules/spam/utils.py

```diff
@@ -83,15 +84,19 @@ def check_and_handle_spam(community=None, deposit=None):
                 'ZENODO_SPAM_CHECK_TIMEOUT'])
         else:
             spam_proba = 0
+
         if spam_proba > current_app.config['ZENODO_SPAM_THRESHOLD']:
             if not Permission(ActionNeed('admin-access')).can():
                 has_records = RecordsSearch(index='records').query(
                     Q('query_string', query="owners:{}".format(
                         community.id_user))).count()
                 has_communities = Community.query.filter_by(
-                        id_user=community.id_user).count()
+                        id_user=community.id_user).count() - 1
+
                 if not (has_records or has_communities):
                     current_app.config['ZENODO_SPAM_HANDLING_ACTIONS'](
                         community=community, deposit=deposit)
+    except BadRequest:
```

Comment:
Not sure if `werkzeug.exceptions.HTTPException` might be a better choice. E.g. in Zenodo custom errors we sub-class from HTTPException usually.

Reply:
I would avoid this since if for instance the ES query times out we don't want it to be raised.